crypto/crypto.go

```diff
@@ -4,75 +4,77 @@ import (
 	"crypto/rand"
 	"errors"
 	"fmt"
-	"log"
-	"os"
 
 	"golang.org/x/crypto/nacl/secretbox"
 	"golang.org/x/crypto/scrypt"
+	"golang.org/x/sys/unix"
```

Comment:
We'll leave that for another PR. One that addresses building on multiple platforms.